.travis.yml

```diff
@@ -1,75 +1,75 @@
 dist: trusty
 sudo: false
 matrix:
-  include:
```

Comment:
Looks like a lot of these yml files have had 4 space formatting applied, can we revert that? If it's prettier's fault we might have to put new rules in prettier rc.

Reply:
Yeah~ it's triggered by prettier pre-commit hook~
Overrider it to 2 spaces indent.
It seems pettier enforce an `indentation style for sequence in mapping`. 
i.e. 
```yaml
paths:
- .yarn
- "yarn.lock"
```
Will become:
```yaml
paths:
  - .yarn
  - "yarn.lock"
```
See:
https://github.com/prettier/prettier/issues/4723
If that's not desired style, we probably have to add those yaml files to ignore list.

Reply:
Ah right. Yeah I suppose the new style is neater, makes for an ugly diff though 😬